.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: Check Code Formatting
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
```

Comment:
Why do we need workflow to run on push event if we have main blocked for pushing?